examples/run_methods/run_slurm.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
-# Use this hps partition (or other valid partition) for SLAC
-hps-mc-batch slurm -o -q hps -r 1:1 -E $HPSMC_DIR/bin/hps-mc-env.sh -W 5 -d $PWD/scratch -c $PWD/.hpsmc -l $PWD/scratch/logs -S $PWD/scratch slic $PWD/job_slurm.json
 
-# Use this ifarm partition (or other valid partition) for JLab
-# hps-mc-batch slurm -o -q ifarm -r 1:1 -E $HPSMC_DIR/bin/hps-mc-env.sh -W 5 -d $PWD/scratch -c $PWD/.hpsmc -l $PWD/scratch/logs -S $PWD/scratch slic $PWD/job_slurm.json
+# Run a slurm job using mostly default arguments.
+hps-mc-batch slurm slic job_slurm.json
```

Comment:
I would much prefer to have an example here where the default arguments are passed in still to illustrate the available options.

Reply:
> 

That's possible. I can do that in a subsequent PR.

Reply:
I changed this to use more command line arguments, and it ran fine at SLAC.